config/settings/wdb.q

```diff
@@ -23,7 +23,7 @@ mode:`save
                                                                                             //                     save mode process.  When this is triggered it will sort the
                                                                                             //                     data on disk, apply attributes and the trigger a reload on the
                                                                                             //                     rdb and hdb processes
-writedownmode:`default                                                                      // the wdb process can periodically write data to disc and sort at EOD in two ways:
+writedownmode:`partbyattr                                                                      // the wdb process can periodically write data to disc and sort at EOD in two ways:
```

Comment:
What does this mean for standard torq setups if we are changing this? Does it impact how they work?

If we have one called default which is not the default....